packages/pn-personagiuridica-webapp/src/components/Contacts/DigitalContactElem.tsx

```diff
@@ -74,52 +64,34 @@ const DeleteDialog: React.FC<DialogProps> = ({
   confirmHandler,
 }) => {
   const { t } = useTranslation(['common']);
-  const isMobile = useIsMobile();
-  const textPosition = useMemo(() => (isMobile ? 'center' : 'left'), [isMobile]);
 
   const deleteModalActions = blockDelete ? (
-    <Button onClick={handleModalClose} variant="outlined" sx={{ width: isMobile ? '100%' : null }}>
+    <Button onClick={handleModalClose} variant="outlined">
       {t('button.close')}
     </Button>
   ) : (
-    <>
-      <Button
-        onClick={handleModalClose}
-        variant="outlined"
-        sx={isMobile ? { width: '100%', mt: 2 } : null}
-      >
+    [
+      <Button key="cancel" onClick={handleModalClose} variant="outlined">
         {t('button.annulla')}
-      </Button>
-      <Button onClick={confirmHandler} variant="contained" sx={{ width: isMobile ? '100%' : null }}>
+      </Button>,
+      <Button key="confirm" onClick={confirmHandler} variant="contained">
         {t('button.conferma')}
-      </Button>
-    </>
+      </Button>,
+    ]
   );
   return (
-    <Dialog
+    <PnDialog
```

Comment:
Checks the comments left on CodeModal about distance between Title and Content, and Content and Actions